README.md

```diff
@@ -1,5 +1,7 @@
 # Captain
 [![Gitter](https://img.shields.io/gitter/room/DAVFoundation/DAV-Contributors.svg?style=flat-square)](https://gitter.im/DAVFoundation/DAV-Contributors)
+[![license](https://img.shields.io/github/license/DAVFoundation/captain.svg?style=flat-square)](https://github.com/DAVFoundation/captain/blob/master/LICENSE)`
```

Comment:
Hey @bonecrushereb.

Can you remove the extra ` from the end of the line, as well as the extra empty line (line 4) you added?

Cheers!